contributors/design-proposals/add-new-patchStrategy-to-clear-fields-not-present-in-patch.md

```diff
@@ -66,11 +65,53 @@ It indicates how to generate and merge a patch for lists. It could be `merge` or
 
 new tags:
 
-`patchStrategy: replaceKeys`:
+`patchStrategy: "replaceKeys"`:
+
+We introduce a new optional directive `$replaceKeys` to support the new patch strategy.
+
+`$replaceKeys` directive has the following properties:
+- It contains a list of strings.
+- All fields needing to be preserved must be present in the `$replaceKeys` list.
+- The fields that are present will be merged with live object.
+- All of the missing fields will be cleared when patching.
+- All fields in the `$replaceKeys` list must be a superset or the same as the fields present in the patch.
+
+An new patch will have same content as old patch and an additional new directive.
```

Comment:
```diff
-An new patch will have same content as old patch and an additional new directive.
+A new patch will have the same content as the old patch and an additional new directive.
```

I'm not sure how carefully you want this to be proofread. There are some minor syntax issues I'm happy to fix if you think that would be useful.